<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGcpDatabaseDialog</class>
 <widget class="QDialog" name="QgsGcpDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Gcp Coverages / Master Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gcp Database settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gcp Database type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbGcpDatabaseType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Directory of Database</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="leOutputDatabaseDir"/>
        </item>
        <item>
         <widget class="QToolButton" name="tbnOutputDatabaseDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Database_File_Edit">
        <property name="text">
         <string>Filename of Database (&apos;.db&apos;)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mOutputDatabaseFileEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_Database_Convention">
       <property name="wordWrap">
        <bool>true</bool>
       </property>
        <property name="text">
         <string>Naming Conventions:&lt;ul&gt;Gcp Coverages: should end with &apos;.gcp.db&apos;&lt;br/&gt;Gcp Master  : should start with &apos;gcp_master&apos;*&apos;.db&apos;&lt;/ul&gt;Default Srid:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0"  colspan="2">
       <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="cbxCreateSqlDump">
        <property name="text">
         <string>Create Sql-Dump of created Database</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbGcpDatabaseType</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>leOutputDatabaseDir</tabstop>
  <tabstop>tbnOutputDatabaseDir</tabstop>
  <tabstop>mOutputDatabaseFileEdit</tabstop>
  <tabstop>cbxCreateSqlDump</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsGcpDatabaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsGcpDatabaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
